README.md

```diff
@@ -85,37 +85,139 @@ Note that a [Synapse account](https://www.synapse.org/#RegisterAccount:0) is req
 Usage as a library
 ------------------
 
-The Synapse client can be used to write software that interacts with the Sage Bionetworks Synapse repository.
+The Synapse client can be used to write software that interacts with the Sage Bionetworks Synapse repository. More examples can be found in the Tutorial section found [here](https://python-docs.synapse.org/build/html/getting_started/basics.html)
 
-### Example
+### Examples
 
-    import synapseclient
+#### Log-in and create a Synapse object
+```
+import synapseclient
 
-    syn = synapseclient.Synapse()
+syn = synapseclient.Synapse()
+
+## log in using auth token
+syn.login(authToken='auth_token')
+```
+
+#### Sync a local directory to synapse
+This is the recommended way of synchronizing more than one file or directory to a synapse project is through the use of `synapseutils`. Using this library let's us handle scheduling everything required to sync an entire directory tree. Read more about the manifest file format in [`synapseutils.syncToSynapse`](https://python-docs.synapse.org/build/html/articles/synapseutils.html#synapseutils.sync.syncToSynapse)
```

Comment:
Nit: could you rephrase this sentence? It has two verbs.

Reply:
I modified this slightly - Let me know if this new version sounds better:

> This is the recommended way of synchronizing more than one file or directory to a synapse project through the use of `synapseutils`. Using this library allows us to handle scheduling everything required to sync an entire directory tree. Read more about the manifest file format in [`synapseutils.syncToSynapse`](https://python-docs.synapse.org/build/html/articles/synapseutils.html#synapseutils.sync.syncToSynapse)

Reply:
Looks great!